classes/python3-usr.yaml

```diff
@@ -12,6 +13,9 @@ buildSetup: &ref |
             fi
         done
     done
+    for j in "" "lib-dynload" "site-packages" "dist-packages" ; do
+        PYTHONPATH+="${PYTHONPATH:+:}${PYTHON3_INSTALL_DIR}${PYTHON3_DESTLIB}${j:+/$j}"
```

Comment:
that is not very clever. in case we would use a different version of a python package than the system one, this would not work. (e.g. setuptools)

better:
`PYTHONPATH="${PYTHON3_INSTALL_DIR}${PYTHON3_DESTLIB}${j:+/$j}${PYTHONPATH:+:$PYTHONPATH}"

BUT this feature is still missing!!

Reply:
Which feature exactly is missing?

Reply:
this one provided by this pull-request ;)